### PR TITLE
chore: refactor communication health check

### PR DESCRIPTION
**Pull request by MakMuftic**
~Add simplistic support for having multiple cron jobs with a configurable frequency. In addition, set up more appropriate interval for executing communication health checks on relayers.~
Refactor when a communication health check is being called and add a few upgrades to logging.

## Description

- Refactor communication health check to execute every 5 minutes
- Try blockstore file opening every 10s (was 5s)
- Turn on logging for indexing chain events

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in docs.
- [x] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: health/health.go [hunk 1]
```diff
@@ -7,16 +7,13 @@ import (
...
 	"github.com/rs/zerolog/log"
 )
 
 // StartHealthEndpoint starts /health endpoint on provided port that returns ok on invocation
-func StartHealthEndpoint(port uint16, c comm.Communication, h host.Host) {
+func StartHealthEndpoint(port uint16) {
 	http.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
-		log.Debug().Msg("Health endpoint called")
-		go comm.ExecuteCommHealthCheck(c, h.Peerstore().Peers())
+		// log.Debug().Msg("Health endpoint called")
```
Comment (mpetrun5) [inline health/health.go:16]: Why the comment?

File: app/app.go [hunk 2]
```diff
@@ -112,7 +112,7 @@ func Run() error {
 		db, err = lvldb.NewLvlDB(viper.GetString(flags.BlockstoreFlagName))
 		if err != nil {
 			log.Error().Err(err).Msg("Unable to connect to blockstore file, retry in 5 seconds")
```
Comment (mpetrun5) [inline app/app.go:114]: Change then the log :smile:

File: example/cfg/config_evm-evm_1.json [hunk 3]
```diff
@@ -8,7 +8,8 @@
         "path": ".",
         "url": "test.url",
         "encryptionKey": "test-enc-key"
-      }
+      },
+      "commHealthCheckInterval": "24h"
```
Comment (mpetrun5) [inline example/cfg/config_evm-evm_1.json:12]: ?

File: jobs/jobs.go [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	"github.com/ChainSafe/sygma-relayer/comm/p2p"
+	"github.com/libp2p/go-libp2p/core/host"
+	"github.com/rs/zerolog/log"
+	"time"
+)
+
+func StartCommunicationHealthCheckJob(h host.Host) {
+	healthComm := p2p.NewCommunication(h, "p2p/health")
+	for {
+		time.Sleep(5 * time.Minute)
```
Comment (mpetrun5) [inline jobs/jobs.go:14]: Remove the config if you hardcode here

File: example/cfg/config_evm-evm_1.json (hunk 3 above)
Comment (MakMuftic) [reply example/cfg/config_evm-evm_1.json:12]: no need for health check running in local setup / e2e tests

**Review by mpetrun5 — APPROVED** [review]

**Review by P1sar — APPROVED** [review]

**Pull request merged**